.github/workflows/dart.yml

```
@@ -173,20 +172,32 @@ jobs:
       - name: Generate API docs
         run: dart pub global run dartdoc
 
-      - name: Download coverage report
+      - name: Download coverage report and badge
         uses: actions/download-artifact@v3
         with:
           name: coverage-report
           path: coverage-report
 
+      - name: Download badge data
+        uses: actions/download-artifact@v3
```

actionlint (1.7.4)

the runner of "actions/download-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)